src/livereload.js

```diff
@@ -128,7 +128,8 @@ class LiveReload {
       reloadMissingCSS: message.reloadMissingCSS != null ? message.reloadMissingCSS : true,
       originalPath: message.originalPath || '',
       overrideURL: message.overrideURL || '',
-      serverURL: `http://${this.options.host}:${this.options.port}`
+      serverURL: `http://${this.options.host}:${this.options.port}`,
+      pluginOrder: message.pluginOrder
```

Comment:
Possibly naive question: isn't `pluginOrder` something you'd want to provide in the script tag?
Would you change it with individual messages?

Reply:
Actually, I thinking about the same thing yesterday, this should be per-page setting, not per-update. I will change this behaviour.